servicenow/servicenow.js

```diff
@@ -0,0 +1,164 @@
+//Copyright 2020 Google LLC
+//Licensed under the Apache License, Version 2.0 (the "License");
+//you may not use this file except in compliance with the License.
+//You may obtain a copy of the License at
+//    https://www.apache.org/licenses/LICENSE-2.0
+//Unless required by applicable law or agreed to in writing, software
+//distributed under the License is distributed on an "AS IS" BASIS,
+//WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//See the License for the specific language governing permissions and
+//limitations under the License.
+
+
+(function process(/*RESTAPIRequest*/ request, /*RESTAPIResponse*/ response) {        
+
+
+        var writer = response.getStreamWriter();
+
+
+        hdrs = {};
+
+
+                var x=request.body.data;
```

Comment:
let's leave for now.  there are comments already in the code explaining what is going on at various points.

Reply:
We probably should avoid the variable name like this, it can easily confuse users when the code gets more complicated. Let's use a more meaningful variable name.

Reply:
renamed vague variables